rust/worker/src/execution/dispatcher.rs

```diff
@@ -298,7 +346,29 @@ mod tests {
         type Result = ();
 
         async fn handle(&mut self, _message: (), ctx: &ComponentContext<MockDispatchUser>) {
-            let task = wrap(Box::new(MockOperator {}), 42.0, ctx.receiver());
+            // Randomly choose between IO task and other task.
```

Comment:
somewhat prefer we don't overload this test harness - gets harder to disentangle two test cases in case of a failure